README.md

```diff
@@ -59,6 +59,46 @@ reconfigured from the application to redefine what each d-pad, thumbstick, butto
 _**Note**: The new tabs opened using the gamepad navigator will use [Google](https://www.google.com/) as the homepage and
 override the default new tab page. However, it can be configured to open any other website instead of Google._
 
+## Supported Actions
+
+The Gamepad Navigator currently supports the following actions for buttons and triggers:
+
+| Action | Description | Invoker Documentation |
+| :--- | :--- | :--- |
+| **Click** | Perform a click on the focused element when the gamepad button or trigger is pressed. | [click](docs/components/inputMapper.base.md#inputmapperbaseclickvalue) |
```

Comment:
"Perform a click" can be simplified to just "Click".

Reply:
Also, for consistency and brevity, you can leave out "when the gamepad button or trigger is pressed".